features/prog.go

```diff
@@ -234,6 +234,10 @@ func HaveProgramHelper(pt ebpf.ProgramType, helper asm.BuiltinFunc) error {
 }
 
 func haveProgramHelper(pt ebpf.ProgramType, helper asm.BuiltinFunc) error {
+	if ok := helperProbeNotImplemented(pt); ok {
```

Comment:
Does it make sense to add a test?

Reply:
Wasn't sure if this needed one because we essentially would just test if a switch in go works but also felt strange to not add a test given this is a regression. I can add `TestHelperProbeNotImplemented` that calls into `haveProgramHelper` with one test case per prog type that we can't probe for at the moment.

Reply:
Let's do it if it's easy to do. Also serves as a reminder to update the tests if we end up implementing the helpers.